### DISCUSSION TITLE
[NHẬN TASK]

### DISCUSSION THREAD

**Discussion by nguyenxuanquang6789** [Runway-Club/Runway-CV-Builder#64] [Task Delivery]
Tôi muốn nhận task : Ràng buộc input của User trong Create CV

**Comment by nguyentantrung2000** [comment]
Thông tin task:
Tránh tình trạng user nhập những thứ không cho phép Frontend.
Yêu cầu:
Ràng buộc tất cả các input (Nếu cần thiết) trong trang create CV 
Deadline 11:59PM 12/8/2021

**Reply by nguyenxuanquang6789** [reply]
Ok
